test/integration/routing_test.rb

```diff
@@ -1,7 +1,7 @@
 require 'test_helper'
 
 class RoutingTest < ActionDispatch::IntegrationTest
```

Comment:
Just a suggestion, the name an location of this test may be confusing compared to the rails guides. They suggest that routing tests should go inside test/controller and should use assertions like `assert_routing "/#{report.slug}", controller: 'reports', action: 'show'`.

https://guides.rubyonrails.org/testing.html#testing-routes

Reply:
@arielj Good point. I just renamed the class name. This file is not really a routing test but more of an integration test.